OSV_Qt_upload/UploadComponent/UploadComponent.pro

```diff
@@ -98,11 +98,17 @@ DEPENDPATH += $$PWD/../zlib
 win32 {
    # LIBS+= $$OUT_PWD/../libusb/release/libusblib.lib
 }
-else:unix {
+else:macx {
     LIBS+= $$OUT_PWD/../libusb/libusblib.a
     LIBS += -framework CoreFoundation -framework Cocoa -framework IOKit
     INCLUDEPATH += $$PWD/../libusb/include/libusb-1.0
 }
+else:unix {
```

Comment:
lib usb is not used anymore so after the code is merged i will remove this dependency